File: LinkERP.Entity/INV/Inventory/Inventory DebtorPrice/LBS_INV_InventoryDebtorPrice.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LinkERP.Entity.INV.Inventory.Inventory_DebtorPrice
{
   public class LBS_INV_InventoryDebtorPrice: BaseEntity
    {
        public Guid CompanyID { get; set; }
        public string ProductID { get; set; }
        public Guid CustomerID{ get; set; }
        public bool? EnablCostPlusMarkup { get; set; }
        public decimal? MarkupPercentage { get; set; }
        public bool PromotionType { get; set; }
        public decimal? PromotionQuantityLimit { get; set; }
        public decimal? PromotionQuantitySold { get; set; }
      
        public string PromotionDays { get; set; }
        public DateTime? PromotionDateFrom { get; set; }
        public DateTime? PromotionDateTo { get; set; }
        public DateTime? PromotionTimeFrom { get; set; }
        public DateTime? PromotionTimeTo { get; set; }
        public decimal? PromotionalPrice { get; set; }
        public decimal? QuantityBreak1 { get; set; }
        public decimal? QuantityBreak2 { get; set; }
        public decimal? QuantityBreak3 { get; set; }
        public decimal? QuantityBreak4 { get; set; }
        public decimal? QuantityBreak5 { get; set; }
        public decimal? PriceBreak1 { get; set; }
        public decimal? PriceBreak2 { get; set; }
        public decimal? PriceBreak3 { get; set; }
        public decimal? PriceBreak4 { get; set; }
        public decimal? PriceBreak5 { get; set; } 
        public string CustomerCode { get; set; } 
        public string CustomerName { get; set; }
        public Guid PriceChangeReason { get; set; }
        public string MarkupPercentage_text { get; set; }
        public string PromotionalPrice_text { get; set; }
        public string QuantityBreak1_text { get; set; }
        public string QuantityBreak2_text { get; set; }
        public string QuantityBreak3_text { get; set; }
        public string QuantityBreak4_text { get; set; }
        public string QuantityBreak5_text { get; set; }
        public string PriceBreak1_text { get; set; }
        public string PriceBreak2_text { get; set; }
        public string PriceBreak3_text { get; set; }
        public string PriceBreak4_text { get; set; }
        public string PriceBreak5_text { get; set; }
        public string QuantityLimit_text { get; set; }
        public string QuantitySold_text { get; set; }
    }
}

File: LinkERP/ClientApp/src/app/models/inv/lbs-inv-inventorydebtorprice.ts
import { BaseEntity } from "../base-entity";

export class LBSINVInventoryDebtorPrice extends BaseEntity {
    CompanyID:any;
    ProductID:any;
    CustomerID:any; 
    EnablCostPlusMarkup:any;
    MarkupPercentage:any;
    PromotionType:any;
    PromotionDays:any;
    PromotionDateFrom:any;
    PromotionDateTo:any;
    PromotionTimeFrom:any;
    PromotionTimeTo:any;
    PromotionalPrice:any;
    PromotionQuantityLimit:any;
    PromotionQuantitySold:any;
    PriceChangeReason:any='00000000-0000-0000-0000-000000000000';
    QuantityBreak1:any;
    QuantityBreak2:any;
    QuantityBreak3:any;
    QuantityBreak4:any;
    QuantityBreak5:any;
    PriceBreak1:any;
    PriceBreak2:any;
    PriceBreak3:any;
    PriceBreak4:any;
    PriceBreak5:any;
    CustomerCode:any;
    QuantityLimit_text :any;
    QuantitySold_text:any;

}
